api/docs/v2/storefront/index.yaml

```diff
@@ -1100,6 +1144,48 @@ components:
               - $ref: '#/components/schemas/User'
               - $ref: '#/components/schemas/Address'
               - $ref: '#/components/schemas/ShipmentAttributesWithoutRelationsips'
+    CreditCard:
+      required:
+        - data
+        - attributes
+        - relationships
+        - included
+      properties:
+        data:
```

Comment:
@przemosk we need to add `included` here also with Payment Method schema, eg. https://github.com/spree/spree/blob/master/api/docs/v2/storefront/index.yaml#L1092